scripts/crawl.py

```diff
@@ -4,10 +4,10 @@
 import os
 import sys
 import traceback
-sys.path.append(os.path.abspath(os.path.join(os.path.expanduser("~"), "conp-dataset")))
-sys.path.append(os.path.join("/tmp", "conp-dataset"))
+sys.path.append(os.path.abspath("/data/giulia_crawler/conp-dataset"))
```

Comment:
Revert this change to 
`sys.path.append(os.path.abspath(os.path.join(os.path.expanduser("~"), "conp-dataset")))	`